### DISCUSSION TITLE
Welcome to zwavejs2mqtt Discussions!

### DISCUSSION THREAD

**Discussion by robertsLando** [zwave-js/zwave-js-ui#62] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by jarrettv** [comment]
First of all great job! Love where this is at already. As someone new to zwave2mqtt and mqtt in general the config can be a little challenging even after reading the documentation. I also see opportunity for other UX improvements. I'm interested helping improve both of these areas. BTW I found this project via reddit and haven't seen it on the official forums.

**Reply by robertsLando** [reply]
Welcome @jarrettv ! 

Please join our slack channel we can speak about this :) 

https://join.slack.com/t/zwave2mqtt/shared_invite/zt-8ns655f6-d407vtI~KjU~1z11jyaQ9Q
